model/project_ref.go

```diff
@@ -3014,21 +3014,42 @@ func (p *ProjectRef) GetProjectSetupCommands(opts apimodels.WorkstationSetupComm
 	return cmds, nil
 }
 
+func updateNextRunTime(baseTime time.Time, definition *PeriodicBuildDefinition) (time.Time, error) {
```

Comment:
nit: should call this `getNextRunTime` since it's not actually updating the next run time in the periodic build definition.